src/solvers/dgsem_p4est/subcell_limiters_3d.jl

```diff
@@ -29,7 +29,7 @@
                                                     mesh, i, j, k, element)
             var = u[variable, i, j, k, element]
             if var < 0
-                error("Safe low-order method produces negative value for conservative variable $variable. Try a smaller time step.")
+                throw(ArgumentError("Safe low-order method produces negative value for conservative variable $variable. Try a smaller time step."))
```

Comment:
I think this is mis-using [`ArgumentError`](https://www.jlhub.com/julia/manual/en/function/ArgumentError), as this is compute on-the-fly, rather than put into a function.

Reply:
Ah yes. Saw it below. Will do that